pt.ipl.isel/programmes/leic/2020-2021-2/timetable.yml

```diff
@@ -10,6 +10,7 @@ programme:
 calendarTerm: "2020-2021-2"
 classes:
 - acr: "AApl"
+  curricularTerm: 4
```

Comment:
The curricularTerm should be specified for each section, since a class may have sections that are from different semesters.

For example, this class is currently being lectured in both 4th and 6th semesters.

Reply:
fixed